### PR TITLE
Run ci_check instead of individual `cargo` commands

### PR DESCRIPTION
**Pull request by curiousleo**
Brings GitHub actions CI closer to what Travis does right now.

## Checklist

- [ ] Updated the documentation (code documentation, command help, ...)
- [ ] Tested the change (unit or integration tests)
- [ ] Amended the changelog in `release.nix` (see `release.nix` for instructions)

### CODE REVIEW

**Comment by curiousleo** [comment]
@Profpatsch any idea what is going on here? https://github.com/target/lorri/pull/430/checks?check_run_id=800250471#step:7:7

If that doesn't ring a bell, I'll investigate tomorrow.

**Comment by Profpatsch** [comment]
Some `?` blows up.

We should add a flag that makes lorri go into “don’t be user-friendly” mode and just print the crash log.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -60,6 +54,10 @@ jobs:
             signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
       - name: Build
         run: nix-build
+      - name: Install
+        run: nix-env -i ./result
+      - name: Test self-upgrade (TODO)
+        run: echo "lorri self-upgrade local $(pwd)"
```
Comment (curiousleo) [inline .github/workflows/ci.yml:60]: `lorri self-upgrade local $(pwd)` blows up right now, I'll deal with that in a separate PR.

Comment (curiousleo) [reply .github/workflows/ci.yml:60]: See #432.

**Comment by curiousleo** [comment]
@Profpatsch tests passed -- is this good to go?

**Pull request merged**